spec/reek/cli/input_spec.rb

```diff
@@ -35,9 +39,20 @@ def argv; end
             and_call_original
         end
 
+        let(:configuration) { test_configuration_for(SAMPLES_PATH.join('configuration/with_excluded_paths.reek')) }
+        let(:path_excluded_in_configration) do
+          SAMPLES_PATH.join('source_with_exclude_paths/ignore_me/uncommunicative_method_name.rb')
+        end
+
         it 'should use working directory as source' do
           expect(subject.sources).to_not be_empty
         end
+
+        it 'should use configuration for excluded paths' do
+          expect(subject).to receive(:configuration).
```

Comment:
Mocking the object under test is not recommended. I would move this test to OptionInterpreter, or use a different implementation of `#configuration` in the dummy class here, that allows to actually pass in the value in the initializer.